store/orm/orm_test.go

```diff
@@ -116,6 +116,36 @@ func TestORM_JobRunsFor(t *testing.T) {
 	assert.Equal(t, []string{jr2.ID, jr1.ID, jr3.ID}, actual)
 }
 
+func TestORM_JobRunsSortedFor(t *testing.T) {
+	t.Parallel()
+
+	store, cleanup := cltest.NewStore()
+	defer cleanup()
+
+	includedJob := cltest.NewJobWithWebInitiator()
+	require.NoError(t, store.CreateJob(&includedJob))
+
+	excludedJob := cltest.NewJobWithWebInitiator()
+	require.NoError(t, store.CreateJob(&excludedJob))
+
+	i := includedJob.Initiators[0]
+	jr1 := includedJob.NewRun(i)
+	jr1.CreatedAt = time.Now().AddDate(0, 0, -1)
+	require.NoError(t, store.CreateJobRun(&jr1))
+	jr2 := includedJob.NewRun(i)
+	jr2.CreatedAt = time.Now().AddDate(0, 0, 1)
+	require.NoError(t, store.CreateJobRun(&jr2))
+	jr3 := excludedJob.NewRun(excludedJob.Initiators[0])
+	jr3.CreatedAt = time.Now().AddDate(0, 0, -9)
+	require.NoError(t, store.CreateJobRun(&jr3))
+
+	runs, count, err := store.JobRunsSortedFor(includedJob.ID, orm.Descending, 0, 100)
+	assert.NoError(t, err)
+	require.Equal(t, 2, count)
+	actual := []string{runs[0].ID, runs[1].ID}
+	assert.Equal(t, []string{jr2.ID, jr1.ID}, actual)
+}
```

Comment:
You've added a where clause above, but this test doesn't seem to test filtering on the job spec id, am I missing it?

Reply:
`jr3` is a run built off of `excludedJob `, and doesn't show up in the final assert.

Reply:
I'll rename `jr3` to `excludedJobRun`.